frontend/src/app/pages/login-page/login-page.component.html

```diff
@@ -1,13 +1,16 @@
 <div class="row">
-    <header class="col-sm-12 col-lg-4">
-        <div class="logo">
-            <img src="assets/icon/medtagger_logo.svg"/>
+    <header class="header col-sm-12 col-lg-4">
+        <div class="header-logo">
+            <img src="../../../assets/icon/medtagger_logo.svg"/>
```

Comment:
Path like that will cause trouble when MedTagger will be run from a subdirectory (mentioned and fixed it in #362), I'd suggest setting it back to `assets/icon/medtagger_logo.svg` ;)

Reply:
@jszemplinski is right, I have wanted to write the same thing.